operator/pkg/ingress/annotations/annotations.go

```diff
@@ -185,3 +187,47 @@ func GetAnnotationTLSPassthroughEnabled(ingress *networkingv1.Ingress) bool {
 
 	return boolVal
 }
+
+// GetAnnotationEnforceHTTPSEnabled retrieves the EnforceHTTPS annotation's value.
+// This uses a string rather than a bool because the empty string means "unset".
```

Comment:
why not using `*bool` instead? IMO it would be more expressive - and easier to use on the consumer side.

below: 

> If the annotation is unset, this returns "" (the empty string).

->

> If the annotation is unset, this returns `nil`.

Reply:
🤦 yeah, that's much better.